components/motor/gpio/motor_encoder.go

```diff
@@ -449,7 +449,11 @@ func (m *EncodedMotor) GoFor(ctx context.Context, rpm, revolutions float64, extr
 		return err
 	}
 
-	return m.opMgr.WaitTillNotPowered(ctx, time.Millisecond, m)
+	if err := m.opMgr.WaitTillNotPowered(ctx, time.Millisecond, m, m.Stop); err != nil {
```

Comment:
can just do `return m.opMgr.WaitTillNotPowered(ctx, time.Millisecond, m, m.Stop)` in all these places

Reply:
made this one line